### DISCUSSION TITLE
Use exit node

### DISCUSSION THREAD

**Discussion by JS-hacs** [hassio-addons/app-tailscale#409] [Ideas]
Please add the ability to use an exit node (not the already implemented advertise as exit node)

**Comment by lmagyar** [comment]
TS tests for running in a HA add-on, and disables this option: https://github.com/tailscale/tailscale/blame/46505ca338797ed4a4b902afd1f989c783000475/envknob/featureknob/featureknob.go#L62-L64

**Reply by lmagyar** [reply]
After the general ban on exit node and ssh in case of HA is removed from TS (https://github.com/tailscale/tailscale/pull/16263), we can make it configurable.

I've made a PR, it is draft, because the changes are not relesed by TS yet. https://github.com/hassio-addons/addon-tailscale/pull/517
